crates/matrix-sdk/tests/integration/widget.rs

```diff
@@ -181,6 +181,35 @@ async fn test_negotiate_capabilities_immediately() {
     assert_matches!(driver_handle.recv().now_or_never(), None);
 }
 
+static HELLO_EVENT: Lazy<JsonValue> = Lazy::new(|| {
+    json!({
+        "content": {
+            "body": "hello",
+            "msgtype": "m.text",
+        },
+        "event_id": "$msda7m0df9E9op3",
+        "origin_server_ts": 152037280,
+        "sender": "@example:localhost",
+        "type": "m.room.message",
+        "room_id": &*ROOM_ID,
+    })
+});
```

Comment:
This looks like this is code that's been just moved, so a good candidate for a separate commit with only code motion.